papers/ring_vrf/abstract-short.tex

```diff
@@ -1,35 +1,35 @@
-which works by adjusting a Groth16 trusted setup to hide public inputs
-when rerandomizing the Groth16.  We then build ring VRFs that amortizes
-expensive ring membership proofs across many ring VRF signatures.
-%
-Incredibly, our ring VRF needs only eight $\mathcal{G}_1$ and two
-$\mathcal{G}_2$ scalar multiplications, making it the only ring signature
-with performance competitive with group signatures.
+%We need a motivation or a reason why we build/need zero-knowledge cont. technique. So I suggest the following explanation. In the previous version, the connection was unclear.
+A signer in our ring VRF construction generates two proofs: one for the correct evaluation of the message and one for the ring membership. We succeed to have the former with discrete logarithm equality (DLEQ) proof. However, the latter requires a more complex relation. Therefore,  we demonstrate our zero-knowledge continuation technique for the ring membership proof.  Our technique lets a signer reuse the same ring membership proof with different messages but the same ring. 
+%NOTE Is the next sentence clear for everyone. Maybe we should restate it
+In a nutshell, it works by adjusting a Groth16 trusted setup to hide public inputs when rerandomizing the Groth16.  
+Our optimization reduces the prover's computations to eight scalar multiplications in the first group and two scalar multiplications in the second group of the bilinear pairing used in Groth16.
+%NOTE: I would remove the claim below because we don't have any comparison related to group signatures in the paper.
+%, making it the only ring signature
+%with performance competitive with group signatures.
 
 \eprintsmallskip
-Ring VRFs produce a unique identity for any given context but remain
+Ring VRF has functional real-world applications.
+A ring VRF signer produces a unique identity for any given context via signed message but remains
 unlinkable between different contexts.  These unlinkable but unique
-pseudonyms provide a far better balance between user privacy and service
-provider or social interests than attribute based credentials like IRMA credentials.
-
-\eprintsmallskip
-Ring VRFs support anonymously rationing or rate limiting resource
-consumption that winds up vastly more flexible and efficient than
-purchases via money-like protocols.
-
-\eprintsmallskip
-We define the security of ring VRFs in the universally composable (UC) model and show that our protocol is UC secure.
+pseudonyms provide a better balance between user privacy and service provider or social interests than attribute-based credentials.
+%NOTE 'a far better' is a strong claim that if we write it we should strongly backup. Also IRMA is not something very well known, so it is better to remove it from here and talk in intro since it does not give much information
+%a far better balance between user privacy and service
+%provider or social interests than attribute-based credentials like %IRMA credentials.
```

Comment:
It's backed up in the intro, imo.  IRMA is going to make GDPR violations more common, like I says in the intro.

"Attribute-based" "credential" has 10x more google hits than "universal composibility" "model".  "I Reveal My Attributes" is not how IRMA is usually named, but alone it has 1/4th as many hits.

Reply:
We are not in a competition of which one has more hits. It is very irrelevant. The issue is that IRMA is not very well known if you are not in this field. So, if someone reads this abstract and see IRMA, the first thing that they do is search it. When I search IRMA in google, I see these results in order: [IRMA - The Initiative for Responsible Mining Assurance](https://responsiblemining.net/), [Hurricane Irma - Wikipedia](https://en.wikipedia.org/wiki/Hurricane_Irma),[IRMA](https://irma.app/?lang=en), [IRMA | Institute of Rural Management Anand, Gujarat, India](https://irma.ac.in/), [IRMA – Irish Recorded Music Association](https://www.irma.ie/).... Which one do you mean should be clear in abstract.